<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>872</width>
    <height>736</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>861</width>
      <height>681</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="checkin_tab">
     <attribute name="title">
      <string>Check-in</string>
     </attribute>
     <widget class="QTableWidget" name="player_table">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>841</width>
        <height>511</height>
       </rect>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <column>
       <property name="text">
        <string>Player</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Region</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Checked in At</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Checked In</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="import_btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>540</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Import
Tournament</string>
      </property>
     </widget>
     <widget class="QPushButton" name="create_btn">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>540</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Create
Tournament</string>
      </property>
     </widget>
     <widget class="QPushButton" name="load_btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>590</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Load
Previous</string>
      </property>
     </widget>
     <widget class="QPushButton" name="start_btn">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>540</y>
        <width>131</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>START
TOURNAMENT!!</string>
      </property>
     </widget>
     <widget class="QLabel" name="status_label">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>590</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Tournament Status - </string>
      </property>
     </widget>
     <widget class="QLabel" name="pcount_label">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>610</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Player Count - </string>
      </property>
     </widget>
     <widget class="QPushButton" name="padd_btn">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>540</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Player</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pedit_btn">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>540</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Edit Player</string>
      </property>
     </widget>
     <widget class="QPushButton" name="premove_btn">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>540</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove Player</string>
      </property>
     </widget>
     <widget class="QPushButton" name="tedit_btn">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>540</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Edit
Tournament</string>
      </property>
     </widget>
     <widget class="QLabel" name="current_url">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>590</y>
        <width>311</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Current URL: temp</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="bracket_tab">
     <attribute name="title">
      <string>Bracket</string>
     </attribute>
     <widget class="QTabWidget" name="groups_tab">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>19</y>
        <width>851</width>
        <height>551</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="group_a">
       <attribute name="title">
        <string>Group A</string>
       </attribute>
       <widget class="QScrollArea" name="group_a_scroll_area">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>841</width>
          <height>521</height>
         </rect>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>839</width>
           <height>519</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="group_b">
       <attribute name="title">
        <string>Group B</string>
       </attribute>
       <widget class="QScrollArea" name="group_b_scroll_area">
        <property name="geometry">
         <rect>
          <x>-1</x>
          <y>-1</y>
          <width>841</width>
          <height>521</height>
         </rect>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>839</width>
           <height>519</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="group_finals">
       <attribute name="title">
        <string>Finals</string>
       </attribute>
       <widget class="QScrollArea" name="group_finals_scroll_area">
        <property name="geometry">
         <rect>
          <x>-1</x>
          <y>-1</y>
          <width>851</width>
          <height>521</height>
         </rect>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>849</width>
           <height>519</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="finish_btn">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>580</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Finish
Tournament</string>
      </property>
     </widget>
     <zorder>finish_btn</zorder>
     <zorder>groups_tab</zorder>
    </widget>
    <widget class="QWidget" name="leaderboard_tab">
     <attribute name="title">
      <string>Leaderboard</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="settings_tab">
     <attribute name="title">
      <string>Settings</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>90</y>
        <width>231</width>
        <height>221</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="labels">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Challonge Username</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Challonge API</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Google Sheet ID</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="save_button">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>320</y>
        <width>111</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="username_text">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>100</y>
        <width>229</width>
        <height>51</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Fira Sans Semi-Light'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="api_text">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>180</y>
        <width>229</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="sheetid_text">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>250</y>
        <width>229</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="saved">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>410</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Saved Config</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>872</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
